### DISCUSSION TITLE
IDE upload speed/stability

### DISCUSSION THREAD

**Discussion by RadekVoltr** [earlephilhower/arduino-pico#432] [Q&A]
Wonder if I have something wrong or it is some issue.

Playing with arduino-pico vs arduino embed and I like more community version. But struggling with upload.

Embed version seems much seamless. Port reset is faster, UF2 drive shows much faster and upload to it is almost instantaneous. 

Arduino-Pico version takes more time, much often end up in situation that upload fails and keep Pico in state that cable unplug and manual bootsel is only possible way.

Is it possible that something is wrong on my side or it is "feature" and we have to live with it ?

**Comment by Andy2No** [comment]
Are you using a Mac, Windows or Linux?  Which version of the OS?

In Windows 7, it's necessary to use Zadig, to set the driver type.  Maybe that's useful for Windows 10 too, I don't know:

See issue #96 (<--click the number)

**Reply by RadekVoltr** [reply]
Windows 10 21H2 19044.1469. Zadig is not used as serial port is properly identified by Windows as USB serial
